db/seeds.rb

```diff
@@ -5,3 +5,22 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+require "./db/data"
+
+local_goals = SeedData::GOALS
+
+local_goals.each do |goal|
+  goal_in_db = Goal.new()
+  goal_in_db.id = goal[:id]
+  goal_in_db.actions = goal[:actions]
+  goal_in_db.description = goal[:description]
+  goal_in_db.evidence = goal[:evidence]
+  goal_in_db.score = goal[:score]
+  goal_in_db.tag = goal[:tag]
+  goal_in_db.title = goal[:title]
+
+  goal_in_db.save
```

Comment:
What do you want to happen in the case of validation errors? Right now `.save` will return false and this will be silently ignored. Would [`save!`](https://apidock.com/rails/ActiveRecord/Base/save%21) be preferable?